Client/mods/deathmatch/logic/CClientPlayer.h

```diff
@@ -136,6 +139,8 @@ class CClientPlayer final : public CClientPed
     unsigned long  m_ulTick;
     bool           m_bDoExtrapolatingAim;
 
+    std::uint8_t   m_scriptDebugLevel;
```

Comment:
One more thing. This variable isn't initialized by default. This can be done right here: `m_scriptDebugLevel{}`;

Reply:
Thx